ches/emailHandler.ts

```diff
@@ -33,14 +37,14 @@ export const sendNewRequestEmails = async (formData: PrivateCloudCreateRequestBo
 export const sendRequestApprovalEmails = async (request: PrivateCloudRequestWithRequestedProject) => {
   const email = render(RequestApprovalTemplate({ request }), { pretty: true });
   try {
-    await chesService.send({
+    await sendEmail({
       body: email,
       // For all project contacts. Sent when the project set deletion request is successfully submitted
       to: [
         request.requestedProject.projectOwner.email,
         request.requestedProject.primaryTechnicalLead.email,
         request.requestedProject.secondaryTechnicalLead?.email,
-      ],
+      ].filter(Boolean),
```

Comment:
this `filter` is unnecessary in favor of `safeEmails` function inside send function https://github.com/bcgov/platform-services-registry/pull/1306/files#diff-47dbc6dc1d6ea3dc3cd61dd971bc213868f28f32ef8d3618d38b1585ad98981eR90